includes/functions-general.php

```diff
@@ -593,8 +593,8 @@ function wpas_tickets_dropdown( $args = array(), $status = '' ) {
 	$options = '';
 
 	foreach ( $tickets as $ticket ) {
-		$is_selected = isset($args['selected']) && $args['selected'] == $ticket->ID ? true : false;
-		$options .= "<option value='$ticket->ID'".($is_selected ? ' selected="selected"' : '').">$ticket->post_title</option>";
+		$is_selected = isset( $args['selected'] ) &&  $ticket->ID === $args['selected'] ? true : false;
```

Comment:
Now that I have another look at it, I realize that it would be best to use WP's helper function for adding the select attribute [`selected()`](https://codex.wordpress.org/Function_Reference/selected). Sorry I didn't think about it earlier.